processing/save_read.py

```diff
@@ -55,4 +55,12 @@ def read_files(spectrometer, files):
 
     else:
         raise ValueError('Unknown spectrometer type')
+
+    # fix comma separated decimals (stored as strings)
+    for col in df.columns:
+        try:
+            df.loc[:, col] = df[col].str.replace(',', '.').astype(float)
```

Comment:
A jeżeli jakiś sprzęt zapisuje pliki w ten sposób, że przecinek jest separatorem tysięcznym to co wtedy? Czasami tak jest, ze masz 1,000,000.123123